rwengine/src/engine/SaveGame.cpp

```diff
@@ -1313,13 +1313,18 @@ bool SaveGame::getSaveInfo(const std::string& file, BasicState* basicState) {
 std::vector<SaveGameInfo> SaveGame::getAllSaveGameInfo() {
     using namespace boost::filesystem;
 
-    // TODO consider windows
+#if defined(RW_WINDOWS)
+    auto homedir = getenv("USERPROFILE");
+    const char gameDir[] = "\\My Documents\\GTA3 User Files";
```

Comment:
Example of the SH* API: https://github.com/paulsapps/alive/blob/a87957409b574e5a12ee85b9a9a1c0de91255f77/include/resourcemapper.hpp#L466